### ISSUE TITLE
bug: still asking to position mod

### ISSUE THREAD

**Issue by laredson** [kldzj/pzmod#16]
**Describe the bug**
While installing mods from list, it has moved forward a step... now doesn't ask to choose a module if the mod has only one modID, but still asks where in the priority list should be placed.
As this game has little to no requirements in mod priority, this seems more of a bug or unintended functioning

**To Reproduce**
Try to install from a collection

**Expected behavior**
Never ask where to place mod priority, only if the mod has several modules inside.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 2.0.1

**Additional context**
Also it should install all maps to the start of the list without asking. there's only a few mods that need to me placed at the very start of the list before every other mod, and those are RV interior, basements and such mods adding further "minihouses" for use in vehicles. If we are installing it we'll know those should be up. Anyways, they should work even it not placed on top, just before muldraug that should be always the last

**Comment by kldzj** [comment]
> Never ask where to place mod priority, only if the mod has several modules inside.

There are mods that need to be placed in the very beginning, I don't think there's a good solution to this problem.

**Comment by kldzj** [comment]
I can't recite any specific mods right now, but I remember there being a few mods that I used that required early loading.

Not sure how to feel about changing the default behavior further, but I'll be adding toggles for this.

**Comment by laredson** [comment]
> I might just add toggles when starting to add from a collection that would allow this behavior.

I think it would work something like that, but default to autoplace down for mods, up for maps.
Now you mention it, yes... I miss an option to reorder ModID and map. What about just don't add a toggle, but add a reorder function to change those few mods that would need certain places

**Comment by kldzj** [comment]
> Now you mention it, yes... I miss an option to reorder ModID and map. What about just don't add a toggle, but add a reorder function to change those few mods that would need certain places

Current flow for that would be to remove and then add again, but I definitely see the benefit of this.

**Comment by laredson** [comment]
> I can't recite any specific mods right now, but I remember there being a few mods that I used that required early loading.

yeah you're right, even if there weren't any atm, in some moment it would appear something that's be needing to be placed somewhere to work.
My point is that there are so little mods that would need that, and even those that explicitly say where to put, in the end they result in be working even if not placed. Somehow PZ manages to load all mods, I think it loads them twice or three times if it detect an error while loading server.

**Comment by kldzj** [comment]
Yeah, I quite like your proposal. It'd make adding mods, especially from collections much easier.

Related/closes #6

**Comment by laredson** [comment]
> Current flow for that would be to remove and then add again, but I definitely see the benefit of this.

mmm I imagine a mod list, as in remove mods, but when you select one it's almost the same as installing it again, as install a mod is adding workshopID and modID... a "reinstall" option would be enough so the program let you choose it's place again

**Comment by laredson** [comment]
> there's only a few mods that need to me placed at the very start of the list before every other mod, and those are RV interior, basements and such mods

I'm think out of the box while saying this, but...
As there's only a few mods to be placed up in maps, maybe you're able to just mark those mods? Not by the API ofc.. but a small db adding those mods that ask to be placed in their description. I would provide all ID that I would find to be needed of placing
